docs/docs/Contributing/contributing-how-to-contribute.mdx

```diff
@@ -83,23 +101,22 @@ The Langflow frontend is served at `http://localhost:7860`.
 </details>
 
 <details>
-<summary>Run from source with Powershell</summary>
+<summary>Run from source with PowerShell</summary>
 
 To run Langflow from source on Windows, you can use the Langflow repository's included scripts, or run the commands in the terminal.
 
 Do one of the following:
 
 * To install and run Langflow with the included scripts, navigate to the `scripts/windows` directory, and then run the `build_and_run.ps1` file.
 
-* To run Langflow from a Powershell terminal:
+* To run Langflow from a PowerShell terminal:
 
-    1. Run the following commands separately to build the Langflow frontend:
+    1. Build the frontend static files:
 
         ```
         cd src/frontend
         npm install
         npm run build
-        npm run start
         ```
```

Comment:
@italojohnny I don't think we want to start the frontend, just build it, so I removed the last command.